config.yml

```diff
@@ -91,6 +90,21 @@ golang:
       - glide --version
       - gin --version
       - modd --version
+    template_vars:
+      GOLANG_VERSION: 1.11
+  "1.13":
```

Comment:
You need to reference this version in `.travis.yml`: `matrix.include` (or update the existing one if you end up dropping 1.11) or else Travis won't make a job for it.